internal/app/app.go

```diff
@@ -13,8 +14,13 @@ import (
 // Start starts the application server.
 func Start() error {
 	r := mux.NewRouter()
+
+	var conf config.Config
+	conf.Load()
+
 	repo := repository.NewRepository()
-	tokenManager := auth.NewTokenManager("NHbNaO6LFERWnOUbU7l3MJdmCailwSzjO76O")
+	tokenManager := auth.NewTokenManager(conf.PrivateKey)
```

Comment:
Token manager will also need a public key. Private is used for issuing tokens and public is for validating.

Reply:
You'll need both private and public key. One is for signing tokens another for validating.